test/unit/jasmine-boot.js

```diff
@@ -80,6 +80,7 @@ async function initializePDFJS(callback) {
       "pdfjs-test/unit/primitives_spec.js",
       "pdfjs-test/unit/scripting_spec.js",
       "pdfjs-test/unit/stream_spec.js",
+      "pdfjs-test/unit/struct_tree_spec.js",
```

Comment:
These test should work when run in Node.js, right?
If so, please make sure to include this file in https://github.com/mozilla/pdf.js/blob/master/test/unit/clitests.json as well.